src/main/java/org/ow2/proactive/procci/model/occi/metamodel/MixinBuilder.java

```diff
@@ -84,7 +84,7 @@ public MixinBuilder(MixinService mixinService, InstanceService instanceService,
         this.applies = new ArrayList<>();
         for (String apply : Optional.ofNullable(mixinRendering.getApplies()).orElse(new ArrayList<>())) {
```

Comment:
simple lambda, something like this : 

this.applies = mixinRendering.getApplies().stream().map(apply -> InfrastructureKinds.getKind(apply)).collect(Collectors.ToList())